app/helpers/check_answers_helper.rb

```diff
@@ -0,0 +1,23 @@
+module CheckAnswersHelper
+  def get_answered_questions_total(subsection_pages, case_log)
+    questions = []
+    subsection_pages.keys.each do |page|
```

Comment:
Could be wrong but I _think_  this will only work for pages that only have a single question on them? (Which is most pages but not strictly all).

Think you might need to loop over `questions_for_page` as well

Reply:
Can we TDD that out at this stage?

Reply:
Might be worth checking this by writing a test that checks it counts the correct number of questions for a section where some pages have multiple questions per age like household_needs or rent.

Reply:
I think that's roughly the idea I had for checking this. Thanks for the advice 😄